SeQuant/domain/mbpt/models/cc.hpp

```diff
@@ -9,12 +9,17 @@
 namespace sequant::mbpt::sr {
 
 /// derives equations of traditional coupled-cluster method
-class cceqs {
+class cc {
   size_t N, P, PMIN;
 
  public:
-  cceqs(size_t n, size_t p = std::numeric_limits<size_t>::max(),
-        size_t pmin = 1);
+  cc(size_t n, size_t p = std::numeric_limits<size_t>::max(), size_t pmin = 1);
+
+  /// derives similarity-transformed expressions of mpbt::Operators
+  /// @param expr expression to be transformed
```

Comment:
if this is supposed to be a ptr to `mbpt::Operator` you can state this here and better yet adding a precondition (to the code and to the dox) ... presumably something like
```
assert(expr.is<mbpt::Operator>());
```
and
```
/// @pre `expr.is<mbpt::Operator>()`
```

Reply:
`expr` here can be a product or sum of `mbpt::Operator` also. I have added `/// @pre expr should be composed of mbpt::Operators`